### ISSUE TITLE
2x2 page alignment issues 

### ISSUE THREAD

**Issue by dendark128** [keotl/invert-pdf#22]
Hi recently the 2x2 conversion is working very weird. The first 9pages are done properly and from then onwards the pages looks very small. Here in the attached file you can see after the page 8 all the images in each page looks very small compared to that from pages 1-8. I converted a 236 pages pdf document to 2x2 for printing purpose. Please look into this issue.
[2x2.pdf](https://github.com/keotl/invert-pdf/files/9997638/2x2.pdf)

**Comment by keotl** [comment]
Hi there, does it occur when only converting to 2x2 without inverting?

**Comment by dendark128** [comment]
yes tried without inverting.. its still the same.. also tried reducing the pages.. did a 63page 2x2 conversion.. same result.. now from my experiments i can say it happens after page 9 so if i do 2x2 of single pdfs then there is no problem cuz single pdfs doesnt contain many pages. So idk whats the issue here but i also tried doing 2x2 from another website that is https://pdfresizer.com/multipage  .. same issues i am facing.. maybe the issue is with my source files although i will try converting pdfs from other sources and tell you. but i have been using this method since months nothing happened till now..
